### PR TITLE
[doc] Netex FR: adding transfers

### PR DESCRIPTION
**Pull request by papailio**

### CODE REVIEW

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 1]
```diff
@@ -219,6 +219,43 @@ StopPlace/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinat
...
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<GeneralFrame id="FR:GeneralFrame:NETEX_TRANSFER:" version="any">
+	<TypeOfFrameRef ref="FR:TypeOfFrame:NETEX_STRUCTURE:"/>
+	<members>
+		<!-- One SiteConnection for each trasnfer in transfers.txt -->
+		<SiteConnection>
+			<From>
+				<!-- Origin stop of the connection -->
+				<QuayRef />
```
Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]: From the NeTEx France profile, I believe we also need `StopPlaceRef`. You can point to the multimodal `StopPlace` whose ID must be `FR:StopPlace:<parent_station>:` where `parent_station` is the field of the corresponding `StopPoint`.

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 2]
```diff
@@ -219,6 +219,43 @@ StopPlace/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinat
...
+			</From>
+			<To>
+				<!-- End stop of the connection -->
+				<QuayRef />
+			</To>
+			<WalkTransferDuration>
```
Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]: This must be the first element of `SiteConnection`, not the last.

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 3]
```diff
@@ -219,6 +219,43 @@ StopPlace/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinat
...
+## correspondances.xml
+Each connection between two stops in `transfers.txt` produces a `SiteConnection` element with the `From` and `To` nodes of the connection as well as a `WalkTransferDuration` node.
+All `SiteConnection` elements are grouped in a `members` element inside a `GeneralFrame`.
+
+### Top level structure
+Example:
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<GeneralFrame id="FR:GeneralFrame:NETEX_TRANSFER:" version="any">
+	<TypeOfFrameRef ref="FR:TypeOfFrame:NETEX_STRUCTURE:"/>
```
Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]: Maybe `NETEX_RESEAU` which is one of the possible value for the profile NeTEx France.

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 4]
```diff
@@ -219,6 +219,43 @@ StopPlace/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinat
...
 
+## correspondances.xml
+Each connection between two stops in `transfers.txt` produces a `SiteConnection` element with the `From` and `To` nodes of the connection as well as a `WalkTransferDuration` node.
+All `SiteConnection` elements are grouped in a `members` element inside a `GeneralFrame`.
+
+### Top level structure
+Example:
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<GeneralFrame id="FR:GeneralFrame:NETEX_TRANSFER:" version="any">
```
Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]: I would use one of the possible `TypeOfFrame` value for the ID of the frame. `NETEX_RESEAU`?

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 5]
```diff
@@ -219,6 +219,47 @@ StopPlace/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinat
...
+## correspondances.xml
+Each connection between two stops in `transfers.txt` produces a `SiteConnection` element with the `From` and `To` nodes of the connection as well as a `WalkTransferDuration` node.
+All `SiteConnection` elements are grouped in a `members` element inside a `GeneralFrame`.
+
+### Top level structure
+Example:
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<GeneralFrame id="FR:GeneralFrame:NETEX_RESEAU:" version="any">
+	<TypeOfFrameRef ref="FR:TypeOfFrame:NETEX_RESEAU:"/>
```
Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]: I would remove that for now to stay homogeneous with the rest of the specification.

Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]: I would remove that for now to stay homogeneous with the rest of the specification.

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 6]
```diff
@@ -219,6 +219,47 @@ StopPlace/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinat
...
+	<members>
+		<!-- One SiteConnection for each trasnfer in transfers.txt -->
```
Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]:
```suggestion
		<!-- One SiteConnection for each transfer in transfers.txt -->
```

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 7]
```diff
@@ -219,6 +219,47 @@ StopPlace/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinat
...
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | ---
+SiteConnection/@id | | | The id is built from the concatenation of the origin and end `stop_id` used in the connection. For the rest of the id, use [id formatting](#id-of-objects).
+SiteConnection/@version | | | Fixed value `any`.
+SiteConnection/From/StopPlaceRef/@ref | | | Id of the multimodal `StopPlace` that contains the origin `Quay` of the connection. See [id formatting](#id-of-objects).
+SiteConnection/From/QuayRef/@ref | transfers.txt | from_stop_id | Id of the origin `Quay` of the connection. See [id formatting](#id-of-objects).
+SiteConnection/To/StopPlaceRef/@ref | | | Id of the multimodal `StopPlace` that contains the end `Quay` of the connection. See [id formatting](#id-of-objects).
+SiteConnection/To/QuayRef/@ref | transfers.txt | to_stop_id | Id of the end `Quay` of the connection. See [id formatting](#id-of-objects).
+SiteConnection/WalkTransferDuration/DefaultDuration | transfers.txt | real_min_transfer_time | Time is given as a [duration](https://en.wikipedia.org/wiki/ISO_8601#Durations) (e.g. PT2M for a transfer time of 2 minutes).
```
Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]: Can you move this line up in the tabular (just after `SiteConnection/@version`).

Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]: Can you move this line up in the tabular (just after `SiteConnection/@version`).

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 8]
```diff
@@ -219,6 +219,47 @@ StopPlace/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinat
...
+				<QuayRef />
+			</To>
+		</SiteConnection>
+	</members>
+</GeneralFrame>
+```
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | ---
+SiteConnection/@id | | | The id is built from the concatenation of the origin and end `stop_id` used in the connection. For the rest of the id, use [id formatting](#id-of-objects).
```
Comment (woshilapin) [inline src/documentation/ntfs_to_netex_france_specs.md]:
```suggestion
SiteConnection/@id | | | The id is built from the concatenation (joined with `_`) of the origin and end `stop_id` used in the connection. For the rest of the id, use [id formatting](#id-of-objects).
```

File: src/documentation/ntfs_to_netex_france_specs.md (hunk 7 above)
Comment (ArnaudOggy) [inline src/documentation/ntfs_to_netex_france_specs.md]:
```suggestion
SiteConnection/WalkTransferDuration/DefaultDuration | transfers.txt | real_min_transfer_time | Time is given as a [duration](https://en.wikipedia.org/wiki/ISO_8601#Durations) (e.g. PT120S for a transfer time of 2 minutes).
```

**Pull request merged**